src/components/Form.js

```diff
@@ -135,8 +135,35 @@ function Form(props) {
 
             // Validate the input for html tags. It should supercede any other error
             _.each(trimmedStringValues, (inputValue, inputID) => {
+
+                const workspaceRegex = inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+                const matches = inputValue.match(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+
+                let isMatch = CONST.WHITELISTED_TAGS.some(r => r.test(inputValue));
```

Comment:
It isn't necessary for VALIDATE_FOR_HTML_TAG_REGEX to have any matches and yet the ignore list may still have a match. Returning early if the input matches with any whitelist tag would allow having invalid inputs along with any valid inputs.

Reply:
I see, that makes sense.